client/src/components/header.jsx

```diff
@@ -64,24 +69,44 @@ class Header extends React.Component {
     })
   }
 
```

Comment:
componentDidMount going to be used later? if not, mind removing?

Reply:
Oh it is there - there's a block of code in this component did mount that does my ajax call. I think github automatically minimizes it because it wants to show you only the parts that I have changed since I last worked on this file.

Reply:
oh lol. my bad